samples/exemplar/Dockerfile

```diff
@@ -1,4 +1,9 @@
 FROM gcr.io/google-appengine/openjdk
 COPY ./build/libs /usr/local/exemplar
 WORKDIR /usr/local/exemplar
+
+RUN groupadd -g 999 appuser && \
+    useradd -r -u 999 -g appuser appuser
+USER 999
```

Comment:
k8s `RunAsNonRoot` requires this to be numeric so it can verify that it is not root.